Add better-stack file (#255)
odidev/ecosystem-dashboard-for-arm: content/commercial_packages/better-stack.md

---
name: Better Stack Uptime
vendor: Better Stack
category: Monitoring/Observability

> description: Better Stack Uptime monitors the availability of your applications and alerts you to any downtime, ensuring uninterrupted service.

Author: ```suggestion
description: Better Stack Uptime monitors the availability of the applications, and alerts for any downtime, ensuring uninterrupted service.
```

product_url: https://betterstack.com/uptime
works_on_arm: true
release_date_on_arm: 2023/06/06

optional_info:
    homepage_url: https://betterstack.com/
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content:

>         vendor_announcement: https://hub.docker.com/layers/betterstack/aws-ecs-fluent-bit/arm64-1/images/sha256-89b99ac3fa15bf4246846a454d3a413e907a38d231ddfe7cb0665fbc7af6330e?context=explore

Author: This isn't an announcement. It's better to keep it empty since we have added other_info.

        official_docs: https://betterstack.com/docs/logs/aws-lambda/

optional_hidden_info:

>     other_info: There hasn't been a specific announcement or blog mentioning ARM64 support, but in the installation steps, it specifies using an [ARM-based instance](https://betterstack.com/docs/logs/aws-ecs/), and they utilize the BetterStack Docker image for ARM64.

Author: ```suggestion
    other_info: Linux/ARM64-specific announcement isn't available. However, in the installation steps, it is mentioned to use an [ARM-based instance](https://betterstack.com/docs/logs/aws-ecs/), and Linux/ARM64 Better Stack docker image can be used for installation.
```
